keyboards/dztech/dz60rgb/keymaps/mekanist/keymap.c

```diff
@@ -50,48 +50,44 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
 
 
 
-void rgb_matrix_layer_helper(uint8_t red, uint8_t green, uint8_t blue, bool default_layer)
-{
-	rgb_led led;
-	
-	for (int i = 0; i < DRIVER_LED_TOTAL; i++) {
-		led = g_rgb_leds[i];
-
-		if (HAS_FLAGS(led.flags, LED_FLAG_MODIFIER)) {
-				rgb_matrix_set_color( i, red, green, blue );
-		}
-	}
+extern led_config_t g_led_config;
+void rgb_matrix_layer_helper (uint8_t red, uint8_t green, uint8_t blue, bool default_layer) {
+  for (int i = 0; i < DRIVER_LED_TOTAL; i++) {
+    if (HAS_FLAGS(g_led_config.flags[i], LED_FLAG_MODIFIER)) {
+        rgb_matrix_set_color( i, red, green, blue );
+    }
+  }
 }
 
 void rgb_matrix_indicators_user(void)
 {
 	uint8_t this_led = host_keyboard_leds();
-	
+
 	if (!g_suspend_state) {
 		switch (biton32(layer_state)) {
 		case _LAYER1:
 			rgb_matrix_layer_helper(0xFF, 0x00, 0x00, false); break;
-			
+
```

Comment:
this isn't any more readable than putting the break on the following line

Reply:
I didn't change the break, it was already like that. What you see is the auto code cleanup removing all the empty spaces on the next line

Reply:
Oh I know, it looks like a drashna'ism. I prefer break to actually be obvious.

Reply:
ya, same way actually. I use to be all over the minimalist white space coding to fit more on the screen. Then I grew up and could afford bigger screens.

Reply:
http://users.csc.calpoly.edu/~jdalbey/SWE/Papers/att_collapse